politeiad/config.go

```diff
@@ -129,11 +137,45 @@ func supportedSubsystems() []string {
 	return subsystems
 }
 
+func parseAndValidateMimeTypes(mimeTypes []string) ([]string, error) {
+	// All MIMES come in index 0, separated by commas
+	var mimes = mimeTypes[0]
+	var parsedMimes []string
+
+	// If it doesn't contain commas, only one mimeType was set
+	if !strings.Contains(mimes, ",") {
+		ext, err := gomime.ExtensionsByType(mimes)
```

Comment:
You don't parse the mime type to make sure its properly formatted.  Why not?

Reply:
Actually the parsing happens at line 147, with the go `mime` package. It's behaving like the discussed above case, where if the package recognizes the mime type, it sets to parsedMimes and is safe to say that it's properly formatted. Otherwise, we issue the warning and don't set it as a valid parsedMime.

```
// If it doesn't contain commas, only one mimeType was set
	if !strings.Contains(mimes, ",") {
		ext, err := gomime.ExtensionsByType(mimes)
		if err != nil {
			return nil, err
		}
		if len(ext) == 0 {
			log.Warnf("MIME type '%v' not recognized", mimes)
		} else {
			parsedMimes = append(parsedMimes, mimes)
		}
	} 
```